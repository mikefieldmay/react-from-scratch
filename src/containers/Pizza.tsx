import React from "react";

import { PizzaImage } from "../components/PizzaImage/PizzaImage";

const Pizza: React.SFC = () => {
  return (
    <div>
      <h1>The Pizza</h1>
      <PizzaImage />
    </div>
  );
};

export default Pizza;
