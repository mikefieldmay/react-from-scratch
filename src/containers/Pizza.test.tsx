import React from "react";
import Pizza from "./Pizza";
import { render } from "@testing-library/react";

describe("<Pizza />", () => {
  test("should render the Pizza component", async () => {
    const { baseElement }: any = render(<Pizza />);
    expect(baseElement).toMatchSnapshot();
  });
});
