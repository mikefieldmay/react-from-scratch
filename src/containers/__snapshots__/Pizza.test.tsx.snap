// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pizza /> should render the Pizza component 1`] = `
<body>
  <div>
    <div>
      <h1>
        The Pizza
      </h1>
      <div
        class="PizzaImage"
      >
        <img
          class="PizzaImg"
          src="[object Object]"
        />
      </div>
    </div>
  </div>
</body>
`;
